README.md

```diff
@@ -44,6 +44,26 @@ For example:
 
 _Add your project ideas below:_
 
+### Kubernetes - Cluster Addons
```

Comment:
```suggestion
### Kubernetes
```

Would make it easier to all k8s projects under this heading :)

Reply:
Good point done!